src/core/components/link/styles.ts

```diff
@@ -50,27 +51,23 @@ export const subdued = css`
 `
 
 export const icon = css`
-	/*
-	FIXME: This makes icons sit vertically centred, but breaks links with long
-	text descriptions onto a separate line to surrounding text
-	 */
-	display: inline-flex;
 	svg {
 		fill: currentColor;
-		position: absolute;
+		vertical-align: middle;
```

Comment:
With this change, the icons rest on the underline or sometimes slightly below it. I know there is some inconsistency in the spacing in the SVGs themeselves, but it would be reassuring if we could improve the way icons look in the example stories.

**Previously**

![Screenshot 2020-10-16 at 15 12 33](https://user-images.githubusercontent.com/5931528/96269429-44f65b00-0fc2-11eb-837f-ed169f05cd07.png)

**Now**

![Screenshot 2020-10-16 at 15 14 33](https://user-images.githubusercontent.com/5931528/96269469-52abe080-0fc2-11eb-9521-bae1a332a74f.png)

I wonder if we could add a bit of artificial spacing using negative margings or something?